### PR TITLE
livecode v1.01

### PR DESCRIPTION
**Pull request by jameshale**

### CODE REVIEW

File: docsets/LiveCode/docset.json [hunk 1]
```diff
@@ -12,6 +12,10 @@
         {
             "version": "1.0.0",
             "archive": "versions/1.0.0/LiveCode.tgz"
-            }
+            },
+         {
+             "version": "1.0.1",
```
Comment (Kapeli) [inline docsets/LiveCode/docset.json]: Newer versions need to be at the top of this list, not bottom.

Comment (jameshale) [reply docsets/LiveCode/docset.json]: Just to check. This is the only place where a new version is specified (apart from its actual presence)?

Comment (Kapeli) [reply docsets/LiveCode/docset.json]: Yep

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**